src/api.js

```diff
@@ -64,6 +64,7 @@ export const getUserById = async (id, token) => {
   const response = await axios.get(`http://localhost:8080/users/${id}`, {
     headers: { Authorization: token }
   });
+  console.log(response)
```

Comment:
Think this isn't needed

Reply:
Yeah, I think i left it in accidentally. I'll remove it